css/index.css

```diff
@@ -1,3 +1,265 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+r older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+.container {
+  width: 880px;
+  /* TB LR */
+  margin: 0 auto;
+  border: 1px solid red;
+  }
+
+/* header */
+header {
+    width:100%;
+    display: flex;
+}
+
+header nav {
+  display: flex;
+  width: 610px;
+  justify-content: space-between;
+  margin-top: 40px;
+}
+
+header  nav a {
+ text-decoration: none;
+    color: #b0b0b0;
+}
+
+header  nav a:hover {
+  text-decoration: underline;
+  color:black;
+}
+
+header .logo {
+    margin: 28px 0 0 91px;
+}
+
+/* Call To Action (CTA) Styles */
+
+.cta {
+    display: flex;
+    margin-top: 72px;
+    justify-content: flex-end;
+}
+
+.cta .cta-text {
+    font-size: 72px;
```

Comment:
Try putting your `font-size` property on the `h1` rather than the .cta-text block. This should make it easier to get the "Innovation On Demand" text and the img to play together.

Reply:
ok, i'll check it out